### ISSUE TITLE
talkback doesn't read menu button in Google play music

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#317]
```
What steps will reproduce the problem?
1. Open Google play music.
2. Navigate to the radio screen.
3. Select any radio station you have created.

What is the expected output? What do you see instead?
According to https://support.google.com/googleplay/answer/3140181?hl=en, there 
should be a menu button for each radio station.  Talkback doesn't read these 
buttons, so you can't manage your radio stations.


What version of the product are you using? On what operating system?
Talkback version 3.4.0 play music 5.1.1109k.772337.  Android 4.1.2.


Please provide any additional information below.


```

Original issue reported on code.google.com by `hittsj...@gmail.com` on 1 Sep 2013 at 6:32

**Comment by GoogleCodeExporter** [comment]
```
This is outside the scope of TalkBack, but I've reported this issue to the 
appropriate team.
```

Original comment by `caseybur...@google.com` on 5 Dec 2013 at 7:37
- Changed state: **Invalid**
